src/app/game-state/main-loop.service.ts

```diff
@@ -71,6 +72,14 @@ export class MainLoopService {
     this.unlockFastSpeed = properties.unlockFastSpeed;
     this.unlockFasterSpeed = properties.unlockFasterSpeed;
     this.unlockFastestSpeed = properties.unlockFastestSpeed;
+    this.topDivider = 10; // For earning bankedticks based on top divider. 
```

Comment:
Took me a bit to understand that 10 = normal speed.

Reply:
Took me a bit too.